opentracing_instrumentation/client_hooks/_dbapi2.py

```diff
@@ -161,13 +161,19 @@ def __init__(self, connection, module_name, connect_params):
             connect_params=connect_params
         )
 
+    def __getattr__(self, name):
+        # Tip suggested here:
+        # https://gist.github.com/mjallday/3d4c92e7e6805af1e024.
+        if name == '_sqla_unwrap':
+            return self.__wrapped__
+
     def __enter__(self):
         with func_span('%s:begin_transaction' % self._module_name):
-            cursor = self.__wrapped__.__enter__()
+            connection = self.__wrapped__.__enter__()
 
-        return CursorWrapper(cursor=cursor,
-                             module_name=self._module_name,
-                             connect_params=self._connect_params)
+        return ConnectionWrapper(connection=connection,
```

Comment:
can you elaborate on this change?

Reply:
Weird. Meant to revert this. Strictly speaking, dbapi2 is not exactly what we are modeling here. Note the example given in the comment above (https://github.com/uber-common/opentracing-python-instrumentation/pull/55/files/d7f5d4d0c0892e98b7a256d0b7933f5814c661de#diff-2b264c7b6b41ea2c077d7e1125a983b2R155):

```
         with MySQLdb.connect(...) as cursor:
             cursor.execute(...)
```

I believe dbapi2 specifies this:

```
         with MySQLdb.connect(...) as conn:
             with conn.cursor() as cursor:
               cursor.execute(...)
```

I was going to make these changes, but SQLAlchemy seems to accept both and I didn't want to break backwards compatibility.